.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
     steps:
       - name: get env
-        run: echo $GITHUB
+        run: echo "${{ toJson(github) }}"
```

Comment:
@danschmidt5189, this just prints out everything when the job runs.  I don't think we really need jq to filter it. do we?

Reply:
No, this looks good to me — no need for `jq`.

Not sure if you saw this, but GitHub published documentation on [printing debugging information](https://docs.github.com/en/actions/learn-github-actions/contexts#example-printing-context-information-to-the-log) and [security hardening for Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections). In light of those, I'd like to see one change — introduce an [intermediate environment variable](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable) instead of printing the result of that expression directly to the console, i.e.:

```yaml
- name: Dump GitHub context
  env:
    GITHUB_CONTEXT: ${{ toJson(github) }}
  run: echo "$GITHUB_CONTEXT"
```

Reply:
👍 I think your example is almost perfect. I think it just needed curly braces. added your suggested change.